build.gradle

```diff
@@ -54,4 +78,16 @@ subprojects {
             html.enabled = true
         }
     }
+
+    test {
+        useJUnitPlatform {
+            excludeTags 'integration'
+        }
+    }
+
+    task integTest(type: Test) {
```

Comment:
I chose to define the `integTest` task for all subprojects instead of just `game-core` because I figured, as code is moved out of `game-core`, other projects will eventually need to run integration tests, as well.